R/scraper.R

```diff
@@ -1,3 +1,13 @@
-scraper <- function(x) {
+#' Function to scrape housing data from a given craiglist url
+#'
+#' @param url The given Craiglist url to scrape the data from.
+#' @param online Whether the data is scraped directly from the url online (default = False). False means the data is scraped from a local HTML file.
+#'
+#' @return tibble A tibble containing listing information such as price, house type and neighborhood.
```

Comment:
Hi Alex. Nice work. When doing the cleaning of the data, I fouund the url contains more information about which city the housing is located in than the neighbourhood column. SO I wonder if we need the neighbourhood column or not.

Reply:
Hi @JuntingHe , thank you for your comments. I see where you are coming from, but since the neighbourhood column does contain some address information in it, I think we can still leave it there to provide more information to end users. Hope this clarifies.